modules/react-components/src/components/input/search-with-filter-labels.tsx

```diff
@@ -154,7 +156,27 @@ export const SearchWithFilterLabels: FunctionComponent<PropsWithChildren<SearchW
                     )
             }
             {
-                (filterLabels && Array.isArray(filterLabels) && filterLabels.length > 0) && (
+                isLoading ? (
+                    <Card.Group style={ { borderRadius: "50px" } }>
+                        {
+                            [ ...Array(numberOfPlaceholderCards) ].map((_, index) => {
```

Comment:
Can't we just use a for loop here?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3653/commits/13fc653e28ee815a2f5323a1157155f5b1e4b3da